localservices/cloudapi/service.go

```diff
@@ -40,6 +40,12 @@ type CloudAPI struct {
 	fabricVLANs   map[int16]*fabricVLAN
 }
 
+type machine struct {
+	cloudapi.Machine
+	NICs        map[string]*cloudapi.NIC `json:"-"`
+	NetworkNICs map[string]string        `json:"-"`
+}
+
```

Comment:
This is only internally, for keeping track of NICs. We don't want to modify `cloudapi.Machine` here because it's the struct that the API returns. I don't want to add extra fields to it to avoid setting the expectation that those will be populated with data from the API.